src/constants.ts

```diff
@@ -11,3 +11,27 @@ export enum ServiceNames {
 }
 
 export const DEFAULT_REALTIME_SLOT_NAME_WITH_WALRUS = 'realtime_rls'
+
+export enum PermissionAction {
+  ANALYTICS_READ = 'analytics:Read',
```

Comment:
Should we just lowercase everything to keep it simple?

Reply:
AWS IAM uses lowercase for the action prefix then UpperCamelCase for the action in documentation, though they are treated case insensitively: https://docs.aws.amazon.com/IAM/latest/UserGuide/reference_policies_elements_action.html. Azure uses lowerCamelCase: https://docs.microsoft.com/en-us/azure/role-based-access-control/role-definitions#actions. `tenant:Sql:CreateTable` is marginally more readable than `tenant:sql:createtable`, so I think we should stick to some form of camel case, we might have other longer action names in the future.

Reply:
Right now we're using case sensitive matching for actions, but it's easy enough to switch to case insensitive matching next time if we wish.

Reply:
soz, yeah, I meant lower camel case, since that treats e.g. the `sql` in `sql:createTable` and `tenant:sql:createTable` consistently. But yeah doesn't matter all that much.